Makefile

```
@@ -112,6 +113,13 @@ endif
 	@echo "Launching via CLI (auto ports)..."
 	@cd $(APPS_DIR) && $(PNPM) -C cli dev -- dev
 
+.PHONY: dev-prod-db
+dev-prod-db: export KANDEV_DATABASE_PATH := $(HOME)/.kandev/data/kandev.db
+dev-prod-db:
```

checkmake (0.3.2)

[warning] Target "dev-prod-db" defined multiple times (lines 117 and 118).

(uniquetargets)